metricbeat/module/windows/service/service_windows.go

```diff
@@ -60,12 +75,34 @@ var serviceStates = map[ServiceState]string{
 	ServiceStopped:         "Stopped",
 }
 
-var serviceStartTypes = map[ServiceStartType]string{
-	ServiceAutoStart:   "Automatic",
-	ServiceBootStart:   "Boot",
-	ServiceDemandStart: "Manual",
-	ServiceDisabled:    "Disabled",
-	ServiceSystemStart: "System",
+type serviceStartType int
+
+const (
+	Boot = iota
+	System
+	Automatic
+	Manual
+	Disabled
+	AutomaticDelayed
+	AutomaticTriggered
+	AutomaticDelayedTriggered
+	ManualTriggered
+)
+
+var serviceStartTypes = [...]string{
```

Comment:
I think using map would be better for maintenance. It's more clear of the mapping and doesn't depend on the orders being the same.